codegens/http/lib/code-http-converter.js

```diff
@@ -1,5 +1,6 @@
 let utils = require('./util'),
-  urlParser = require('url').parse;
+  _ = require('./lodash'),
+  sdk = require('postman-collection');
```

Comment:
We'll probably have to add this to the dependencies in package.json, because these modules should be individually usable too. We usually avoid using any dependencies in these codegens to avoid increasing the size of these modules. But this is a special use case we'll have to add it.

Reply:
Done